vtkCleaverImageToUnstructuredGridFilter.h

```diff
@@ -0,0 +1,107 @@
+/*=========================================================================
+
+  Program:   Visualization Toolkit
+  Module:    vtkCleaverImageToUnstructuredGridFilter.h
+
+  Copyright (c) Ken Martin, Will Schroeder, Bill Lorensen
+  All rights reserved.
+  See Copyright.txt or http://www.kitware.com/Copyright.htm for details.
+
+     This software is distributed WITHOUT ANY WARRANTY; without even
+     the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+     PURPOSE.  See the above copyright notice for more information.
+
+=========================================================================*/
+#ifndef vtkCleaverImageToUnstructuredGridFilter_h
+#define vtkCleaverImageToUnstructuredGridFilter_h
+
+#if defined(_WIN32)
+#define VTKCLEAVER_EXPORT __declspec( dllexport )
+#else
+#define VTKCLEAVER_EXPORT
+#endif
```

Comment:
Remove this and instead include 

```
#include "vtkCleaverImageToUnstructuredGridModule.h" // For export macro
```

Reply:
OK, I thought we had to include cmake code for this to be available, but I see the vtk module takes care of it.

Reply:
> but I see the vtk module takes care of it.

Indeed, with the current approach everything is taking care of and the next step will only be one possible approach for dissemination.

Reply:
fixed.